Source/ACE/Managers/GuidManager.cs

```diff
@@ -45,6 +53,16 @@ public static uint NewPlayerGuid()
         }
 
         /// <summary>
+        /// Returns New Admin Guid
+        /// </summary>
+        /// <returns></returns>
+        public static uint NewAdminGuid()
```

Comment:
This function is clearly not thread-safe, but will likely be used (or try to be used) from a threaded context.  Either make it thread safe with a lock (recommended) or (if your feeling fancy) atomic increment (https://msdn.microsoft.com/en-us/library/dd78zt0c(v=vs.110).aspx, note you'll have to do casting to make this work), or put in the header the exact contexts it can be called from.